tests/unit/callback_tests.py

```diff
@@ -79,9 +83,7 @@ def test_name_or_value_str(self):
 
     def test_name_or_value_unicode(self):
         value = u'Это тест значения'
-        expectation = ('\xd0\xad\xd1\x82\xd0\xbe \xd1\x82\xd0\xb5\xd1\x81\xd1'
-                       '\x82 \xd0\xb7\xd0\xbd\xd0\xb0\xd1\x87\xd0\xb5\xd0\xbd'
-                       '\xd0\xb8\xd1\x8f')
+        expectation = 'Это тест значения'
```

Comment:
test for the canonical str which in Python 2 is an `str` in Python 3, also an `str`